content/flux/v0.x/join-data/_index.md

```diff
@@ -0,0 +1,143 @@
+  Flux supports inner, full outer, left outer, and right outer joins.
+  Learn how to use the `join` package to join two data sets with common values.
+menu:
+  flux_0_x:
+    name: Join data
+weight: 8
+---
+
+The Flux [`join` package](/flux/v0.x/stdlib/join/) lets you perform inner,
+left outer, right outer, and full outer joins.
+Each join method combines from both input streams in a different way.
+Learn how to use the `join` package to join two data sets with common values.
+
+The `join` package supports the following join methods:
+
+{{< flex >}}
+{{< flex-content "quarter" >}}
+<a href="#perform-an-inner-join">
+  <p style="text-align:center"><strong>Inner join</strong></p>
+  {{< svg svg="static/svgs/join-diagram.svg" class="inner small center" >}}
+</a>
+{{< /flex-content >}}
+{{< flex-content "quarter" >}}
+<a href="#perform-a-left-outer-join">
+  <p style="text-align:center"><strong>Left outer join</strong></p>
+  {{< svg svg="static/svgs/join-diagram.svg" class="left small center" >}}
+</a>
+{{< /flex-content >}}
+{{< flex-content "quarter" >}}
+<a href="#perform-a-right-outer-join">
+  <p style="text-align:center"><strong>Right outer join</strong></p>
+  {{< svg svg="static/svgs/join-diagram.svg" class="right small center" >}}
+</a>
+{{< /flex-content >}}
+{{< flex-content "quarter" >}}
+<a href="#perform-a-full-outer-join">
+  <p style="text-align:center"><strong>Full outer join</strong></p>
+  {{< svg svg="static/svgs/join-diagram.svg" class="full small center" >}}
+</a>
+{{< /flex-content >}}
+{{< /flex >}}
+
+- [How join functions work](#how-join-functions-work)
+  - [Input streams](#input-streams)
+  - [Join predicate function (on)](#join-predicate-function-on)
+  - [Join output function (as)](#join-output-function-as)
+- [Perform join operations](#perform-join-operations)
+  {{< children type="anchored-list" filterOut="Troubleshoot join operations" >}}
+- [Troubleshoot join operations](#troubleshoot-join-operations)
+
+## How join functions work
+
+All functions in the `join` package join _two_ streams of tables together based 
+on common values in each input stream.
+
+- [Input streams](#input-streams)
+- [Join predicate function (on)](#join-predicate-function-on)
+- [Join output function (as)](#join-output-function-as)
+
+### Input streams
+
+Each input stream is assigned to the `left` or `right` parameter.
+Input streams can be defined from any valid data source.
+For more information, see:
+
+- [Query data sources](/flux/v0.x/query-data/)
+- Define ad hoc tables with [`array.from()`](/flux/v0.x/stdlib/array/from/)
+
+#### Data requirements
+
+To join two streams of data with the `join` package, each stream must have:
+
+- **One or more columns with common values to join on**.  
+  Columns do not need identical labels, but they do need to have comparable values.
+- **Identical [group keys](/flux/v0.x/get-started/data-model/#group-key)**.  
+  Functions in the `join` package use group keys to quickly determine what tables
```

Comment:
I think it'd be helpful to include this explicitly in procedures as a step, if we don't already

Reply:
This section is linked to in all the procedural docs, but I was trying to avoid repeating it in every single one.